app/javascript/schema.json

```diff
@@ -116,6 +116,32 @@
               },
               "isDeprecated": false,
               "deprecationReason": null
+            },
+            {
+              "name": "tweets",
```

Comment:
This schema update happens after the initial schema is introduced in the other PR. I think it makes sense to just create the backend with your final version of the schema since you're introducing a new feature.